### PR TITLE
feat(console): api resources list page

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

feat(console): api resources list page

## Linear Issue Reference

* LOG-1694

## Testing

resource-api

### CODE REVIEW

File: packages/console/src/pages/ApiResources/index.module.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+.headline {
+  display: flex;
+  justify-content: space-between;
+}
+
+.table {
+  margin-top: _.unit(6);
+  width: 100%;
+
+  tbody tr.clickable {
```
Comment (simeng-li) [inline packages/console/src/pages/ApiResources/index.module.scss:12]: Just wondering where comes the clickable className for tr elements?

Comment (xiaoyijun) [reply packages/console/src/pages/ApiResources/index.module.scss:12]: sorry. I miss it.
Added on the API Resource page.

File: packages/phrases/src/locales/zh-cn.ts [hunk 2]
```diff
@@ -48,6 +48,13 @@ const translation = {
         tranditional: 'Tranditional Web App',
       },
     },
+    api_resources: {
+      title: 'API 资源管理',
```
Comment (gao-sun) [inline packages/phrases/src/locales/zh-cn.ts:52]:
```suggestion
      title: 'API 资源',
```

File: packages/phrases/src/locales/zh-cn.ts [hunk 3]
```diff
@@ -48,6 +48,13 @@ const translation = {
...
+      subtitle: 'Define APIs that you can consume from your authorized applications.',
+      create: '创建新的API',
+      api_name: 'API 名称',
```
Comment (gao-sun) [inline packages/phrases/src/locales/zh-cn.ts:55]: I think we can just leave this in English until we have finalized content

**Review by gao-sun — APPROVED** [review]

File: packages/phrases/src/locales/en.ts [hunk 4]
```diff
@@ -46,6 +46,13 @@ const translation = {
         tranditional: 'Tranditional Web App',
       },
     },
+    api_resources: {
+      title: 'API Resources',
+      subtitle: 'Define APIs that you can consume from your authorized applications.',
+      create: 'Create API',
```
Comment (gao-sun) [inline packages/phrases/src/locales/en.ts:52]: Create API Resource

**Pull request merged**